Core/Core/Extensions/UIImageViewExtensions.swift

```diff
@@ -27,12 +28,7 @@ public struct LoadedImage {
 private var urlHandle: UInt8 = 0
 private var loaderHandle: UInt8 = 0
 
-public protocol ImageLoadingView: class {
-    var frame: CGRect { get }
-    func load(url: URL, didCompleteWith: LoadedImage?, error: Error?)
-}
-
-extension UIImageView: ImageLoadingView {
+extension UIImageView {
```

Comment:
I've also been working on Avatar & loading remote images for swiftui, so we're going to conflict here.